internal/v2/s2av2_e2e_test.go

```diff
@@ -256,7 +284,48 @@ func TestEndToEndUsingFakeS2AOnUDSEmptyId(t *testing.T) {
 	// Finally, start up the client.
 	ctx, cancel := context.WithTimeout(context.Background(), defaultE2ETimeout)
 	defer cancel()
-	runClient(ctx, t, clientS2AAddr, serverAddr, nil)
+	runClient(ctx, t, clientS2AAddr, serverAddr, nil, nil)
+}
+
+func TestGRPCFallbackEndToEndUsingFakeS2AOverTCP(t *testing.T) {
+	fallback.FallbackTLSConfigGRPC = tls.Config{
```

Comment:
Rather than totally overwriting fallback.FallbackTLSConfigGRPC, can we just set the value of the `InsecureSkipVerify` field?